tests/xfstests/run.pm

```diff
@@ -255,7 +255,10 @@ sub tests_from_ranges {
 sub test_run {
     my $test = shift;
     my ($category, $num) = split(/\//, $test);
-    my $cmd = "\n$TEST_WRAPPER '$test' | tee $LOG_DIR/$category/$num; ";
+    my $run_options = '';
+    $run_options = '-nfs' if check_var('XFSTESTS', 'nfs');
+    my $inject_code = get_var('INJECT_INFO', '');
+    my $cmd = "\n$TEST_WRAPPER '$test' $run_options $inject_code | tee $LOG_DIR/$category/$num; ";
```

Comment:
Where is inject_line?

Reply:
They are in the same parameter, e.g. INJECT_INFO='42 do_something'